lib/gateways/rent_charges_gateway.rb

```diff
@@ -0,0 +1,11 @@
+class RentChargesGateway
+  def save(rent_charges)
+    rent_charges.each do |rent_charge|
+      RentCharge.create!(rent_charge)
```

Comment:
This has become slightly ambiguous to read now, as we have two `RentCharge` objects - the Domain one, and the ActiveRecord one. We could namespace all models so this was `Models::RentCharge`, for example?